src/axios/axiosInstance.ts

```diff
@@ -24,7 +25,7 @@ axiosInstance.interceptors.request.use(
       config.headers.Authorization = `Bearer ${accessToken}`;
     }
 
-    if (config.url && config.url.includes(ENDPOINTS.signout) && refreshToken) {
+    if (config.url && config.url === ENDPOINTS.signout && refreshToken) {
```

Comment:
use ?. operator here

Reply:
when I use ?. it says this operator is only for reading properties, not assignment
so I make it like
  if (config.headers) {
        config.headers['refresh-token'] = refreshToken;
      }